pki/util.go

```diff
@@ -136,6 +136,34 @@ func GenerateCACertAndKey(commonName string, privateKey *rsa.PrivateKey) (*x509.
 	return kubeCACert, rootKey, nil
 }
 
+func GetIPHostname(host *hosts.Host) *cert.AltNames {
```

Comment:
can we change the name to express that we are getting alt names for a specific hosts